README.md

```diff
@@ -1 +1,172 @@
+For Google Analytics, [those instructions are here](https://support.google.com/analytics/answer/1008080).
+
+When you find the code snippet, paste it into `404.html` where marked (above the `redirect.js` script).
+
+### Set up custom domain
+
+By default, GitHub Pages will host your redirects website at a url like `your-org.github.io/repo-name/some-link`.
+To turn this into a nicer `your-domain.com/some-link`, you can set up a custom domain.
+
+To do this, [follow the instructions here](https://docs.github.com/en/github/working-with-github-pages/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain).
+In summary:
+
+1. Purchase a domain name from a reputable service.
+2. Point your domain name provider to GitHub Pages using an `A` record.
+   This is slightly different for each company; they should have their own instructions on how to do it.
+3. Set the custom domain field in the "Pages" settings of the _website repo_ (automatically creates a `CNAME` file in the repo).
+4. GitHub should tell you that your site is now being hosted at e.g. `your-domain.com`.
+
+### Add notes
+
+These aren't always necessary or desired, but may help the users and editors of your links.
+
+- In your _redirects repo_, add a big warning to the top of the readme like "_Do not share links here without permission 🔒_", so that people who have access understand that it's privileged info.
+- In your _website repo_, add an `index.html` page with some filler content like "_This website just performs redirects for [YOUR ORG]_", in case people go to the root of the website with no e.g. `/some-link`.
+- In your _website repo_, add a big link to the top of the readme to remind people where your website is hosted, e.g. `your-domain.com` or `your-org.github.io/repo-name`.
+
+### Existing site
+
+If you already have an existing website being hosted with GitHub Pages that you want to incorporate this approach into:
+
+1. Skip forking the _website repo_.
+2. Copy the [`redirect.js` script](https://github.com/CU-DBMI/redirects-website/blob/main/redirect.js) into the **top folder** of your existing website repo.
+3. [Include it](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/The_head_metadata_in_HTML#applying_css_and_javascript_to_html) at the top of your 404 page `<head>`, after your analytics script(s) but before anything else (so the redirection happens immediately).
+
+If your existing website is built and hosted in a different way, this approach would require modification[^3] and might not be appropriate for you.
+
+[^1]: This approach does not _encrypt_ your redirect list, it only _obfuscates_ it. Anyone with some coding knowledge could still figure out all of your redirect lists with some effort. The way this method works, true encryption is impossible because any key needed for decryption would also need to be in the `redirect.js` script, which the attacker would have access to.
```

Comment:
```suggestion
[^1]: This approach does not _encrypt_ your redirect list, it only _obfuscates_ it. Anyone with some coding knowledge could still figure out all of your redirect lists with some effort. Because of the way this method works true encryption is impossible, because any key needed for decryption would also need to be in the `redirect.js` script, to which the attacker would have access.
```